checks.d/kong.py

```diff
@@ -0,0 +1,71 @@
+# stdlib
+import urlparse
+
+# 3rd party
+import requests
+import simplejson as json
+
+# project
+from checks import AgentCheck
+from util import headers
+
+
+class Kong(AgentCheck):
+    """ collects metrics for Kong """
+    def check(self, instance):
+        metrics = self._fetch_data(instance)
+        for row in metrics:
+            try:
+                name, value, tags = row
+                self.gauge(name, value, tags)
+            except Exception:
+                self.log.error(u'Could not submit metric: %s', row)
+
+    def _fetch_data(self, instance):
+        if 'kong_status_url' not in instance:
+            raise Exception('missing "kong_status_url" value')
+        tags = instance.get('tags', [])
+        url = instance.get('kong_status_url')
+
+        parsed_url = urlparse.urlparse(url)
+        host = parsed_url.hostname
+        port = parsed_url.port or 8001
```

Comment:
Indeed, but if a URL is provided without a specific port, wouldn't `requests` try the port 80 ? (Since this port is taken from the `kong_status_url` which is queried afterwards)